Cargo.toml

```diff
@@ -24,19 +24,26 @@ members = [
 
 resolver = "2"
 
-exclude = [
-	"node-graph/gpu-compiler",
-]
+exclude = ["node-graph/gpu-compiler"]
 
 [workspace.dependencies]
-specta = { git = "https://github.com/oscartbeaumont/rspc", rev = "9725ddbfe40183debc055b88c37910eb6f818eae", features = ["glam"] }
+specta = { git = "https://github.com/oscartbeaumont/rspc", rev = "9725ddbfe40183debc055b88c37910eb6f818eae", features = [
+	"glam",
+] }
 xxhash-rust = { version = "0.8.4", features = ["xxh3"] }
 
-[profile.release.package.graphite-wasm]
-opt-level = 3
+
+[profile.dev.package.graphite-editor]
+opt-level = 1
+
+[profile.dev.package.graphene-core]
+opt-level = 1
+
+[profile.dev.package.graphene-std]
+opt-level = 1
 
 [profile.dev.package.graphite-wasm]
-opt-level = 3
+opt-level = 1
```

Comment:
Shouldn't this work by optimizing all deps fully, except our own first-party workspace crates? I see how this explicitly optimizes autoquant, image, etc. but not everything third-party.